go.mod

```diff
@@ -72,6 +72,7 @@ require (
 	github.com/OneOfOne/xxhash v1.2.8 // indirect
 	github.com/agnivade/levenshtein v1.1.1 // indirect
 	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da // indirect
+	github.com/benburkert/pbench v0.0.0-20160623210926-4ec5821845ef // indirect
```

Comment:
dependency from 2016, do we need this or can we copy a function into our code base?